src/utils/i18n/translations/en-US.json

```diff
@@ -6,6 +6,7 @@
   "common.more": "More",
   "common.loading": "Loading",
   "common.select.noResults": "No results",
+  "common.noOptions": "No Options",
```

Comment:
wouldn't this add an inconsistency in the select components? previously we use `No results`